doc/whats_new/upcoming_changes/sklearn.inspection/31693.enhancement.rst

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I don't think this empty line is needed.
Could you also make sure your lines are <88 characters long? Thank you